src/models/discriminant_analysis.jl

```diff
@@ -67,6 +67,14 @@ function MMI.fit(model::LDA, ::Int, X, y)
     return fitresult, cache, report
 end
 
+const ERR_LONE_TARGET_CLASS = ArgumentError(
+    "The number of unique classes in "*
+    "the training target has to be greater than one. It does not "*
+    "suffice to have more than one class in the pool. This is not "*
```

Comment:
```suggestion
    "suffice to have only one class in the pool. This is not "*
```
I'm thinking this is probably a mistake.

Reply:
I've reworded this to make it clearer.

Reply:
```julia
const ERR_LONE_TARGET_CLASS = ArgumentError(
    "The number of unique classes in "*
    "the training target has to be greater than one, even if "*
    "the complete pool contains more than one class. If "*
    "`cov_b=CovarianceEstimation.SimpleCovariance()` this "*
    "restriction on classes does not apply. "
)
```